src/main/cli/cli.c

```diff
@@ -3105,10 +3105,10 @@ static void cliSimplifiedTuning(const char *cmdName, char *cmdline)
 }
 #endif
 
-static void printCraftName(dumpFlags_t dumpMask, const pilotConfig_t *pilotConfig)
+static void printName(dumpFlags_t dumpMask, const pilotConfig_t *pilotConfig)
```

Comment:
Can't this function remain `printCraftName`

Reply:
Guess we only need to retain `MSP_NAME` and `MSP_SET_NAME` commands.

Await testing by @howels